usaspending_api/transactions/delta_models/transaction_search.py

```diff
@@ -462,10 +462,14 @@
         SAA.abbreviation AS awarding_subtier_agency_abbreviation,
         SFA.abbreviation AS funding_subtier_agency_abbreviation,
         COALESCE(transaction_fabs.awarding_office_code, transaction_fpds.awarding_office_code) AS awarding_office_code,
-        COALESCE(transaction_fabs.awarding_office_name, transaction_fpds.awarding_office_name) AS awarding_office_name,
-        COALESCE(transaction_fabs.funding_office_code, transaction_fpds.funding_office_code) AS funding_office_code,
-        COALESCE(transaction_fabs.funding_office_name, transaction_fpds.funding_office_name) AS funding_office_name,
-
+        CASE
+            WHEN COALESCE(transaction_fabs.awarding_office_code, transaction_fpds.awarding_office_code) IS NOT NULL
```

Comment:
I would update the `COALESCE` to take into account the possible case where a value isn't found in the office table, but we do have a name from the raw transaction.